chart/scripts/jupyterhub/config/jupyterhub_profiles.py

```diff
@@ -1123,7 +1123,7 @@ def mutate_pod_spec_for_kernel_gateway(spawner):
     c.JupyterHub.authenticate_prometheus = False
     c.JupyterHub.template_vars = { 'primehub_version': primehub_version }
     c.JupyterHub.logo_file = '/usr/local/share/jupyterhub/static/images/PrimeHub.png'
-    c.PrimeHubSpawner.start_timeout = 60 * 5
+    c.PrimeHubSpawner.start_timeout = int(os.environ.get('SPAWNER_START_TIMEOUT', '300'))
```

Comment:
envs should come from `.Values.primehub.hub.extraEnv`

https://github.com/InfuseAI/primehub/blob/v3.1.1/chart/values.yaml#L495-L508

Reply:
we could add `c.PrimeHubSpawner.http_timeout = os.environ.get('SPAWNER_HTTP_TIMEOUT', '30')` for `creating` timeout

Reply:
However, there is another place to put configuration `.Values.primehub.hub.primehub.spawner.XXX_timeout`, and it is a map not a list, it would be easier to update from helm `--values` (we use `get_primehub_config` function to get values)